.travis.yml

```diff
@@ -5,9 +5,11 @@ env:
   global:
     - CXX=g++-4.8
   matrix:
-    - TEST_SUITE=test
-    - TEST_SUITE=css:test
-    - TEST_SUITE=css:reference
+    - TEST_SUITE='test'
```

Comment:
good question. because they are run in parallel, that would not be possible. but because they are run in parallel, it shouldn't impact how long our build takes.